### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'group'

### ISSUE THREAD

**Issue by arielelkin** [toxinu/pyhn#39]
I seem to have the latest version of `pyhn`:

```
$ pip show pyhn
---
Metadata-Version: 2.0
Name: pyhn
Version: 0.3.7
Summary: Hacker News in your terminal
Home-page: https://github.com/toxinu/pyhn/
Author: Geoffrey Lehée
Author-email: contact@toxi.nu
Installer: pip
License: The MIT License (MIT)
Location: /usr/local/lib/python2.7/site-packages
Requires: urwid, beautifulsoup4, requests
Classifiers:
```

But I'm getting this error when I run it:

```
$ pyhn
Loading stories...
Traceback (most recent call last):
  File "/usr/local/bin/pyhn", line 9, in <module>
    hn_gui.main()
  File "/usr/local/lib/python2.7/site-packages/pyhn/gui.py", line 98, in main
    self.build_interface()
  File "/usr/local/lib/python2.7/site-packages/pyhn/gui.py", line 144, in build_interface
    self.cache_manager.refresh()
  File "/usr/local/lib/python2.7/site-packages/pyhn/cachemanager.py", line 43, in refresh
    stories = self.api.get_top_stories(extra_page=self.extra_page)
  File "/usr/local/lib/python2.7/site-packages/pyhn/hnapi.py", line 353, in get_top_stories
    stories += self.get_stories(source)
  File "/usr/local/lib/python2.7/site-packages/pyhn/hnapi.py", line 260, in get_stories
    story_published_time.append(self.get_published_time(story))
  File "/usr/local/lib/python2.7/site-packages/pyhn/hnapi.py", line 190, in get_published_time
    return results.group()
AttributeError: 'NoneType' object has no attribute 'group'
```

**Comment by DotComing** [comment]
Hi, @arielelkin, I have met the same situation and created the pr #38 , I have explained why, you may hava a look at my pr, correct me if I am wrong.

**Comment by arielelkin** [comment]
Nice one, @DotComing, thanks for letting me know. I'll update this issue once #38 gets merged.

**Comment by toxinu** [comment]
Thank you guys for reporting this bug.
We just need a little modification on @DotComing pull request to merge it.
Then, I'll do a release on Pypi. :smile:

**Comment by toxinu** [comment]
I just released `0.3.8` on Pypi.
Can you give me some feedback guys? :smiley:

**Comment by arielelkin** [comment]
issue gone, nice one @toxinu, thank you.
